MonoGame.Framework/PlatformInfo.Platform.cs

```diff
@@ -0,0 +1,72 @@
+﻿namespace Microsoft.Xna.Framework
+{
+#if ANDROID
+    partial class PlatformInfo
+    {
+        private static MonogamePlatform PlatformGetCurrent()
+        {
+            return MonogamePlatform.Android;
+        }
+    }
+#endif
+
+#if DESKTOPGL
+    partial class PlatformInfo
+    {
+        private static MonogamePlatform PlatformGetCurrent()
+        {
+            return MonogamePlatform.DesktopGL;
+        }
+    }
+#endif
+
+#if IOS && !TVOS
+    partial class PlatformInfo
+    {
+        private static MonogamePlatform PlatformGetCurrent()
+        {
+            return MonogamePlatform.iOS;
+        }
+    }
+#endif
+
+#if TVOS
+    partial class PlatformInfo
+    {
+        private static MonogamePlatform PlatformGetCurrent()
+        {
+            return MonogamePlatform.tvOS;
+        }
+    }
+#endif
+
+#if WEB
+    partial class PlatformInfo
+    {
+        private static MonogamePlatform PlatformGetCurrent()
+        {
+            return MonogamePlatform.Web;
+        }
+    }
+#endif
+
+#if WINDOWS && DIRECTX
+    partial class PlatformInfo
+    {
+        private static MonogamePlatform PlatformGetCurrent()
+        {
+            return MonogamePlatform.Windows;
+        }
+    }
+#endif
+
+#if WINDOWS_UAP
+    partial class PlatformInfo
+    {
+        private static MonogamePlatform PlatformGetCurrent()
+        {
+            return MonogamePlatform.WindowsUniversal;
```

Comment:
You can have only the return statement in the `#if` `#endif` block i.e.
```
partial class PlatformInfo
{
    private static MonoGamePlatform PlatformGetCurrent()
    {
#if DESKTOPGL
        return MonoGamePlatform.DesktopGL;
#endif
        // etc....
    }
}
```

Reply:
I did it that way on purpose, so closed platforms could make their own implementation of PlatformGetCurrent().
Though if I'll be provided closed platforms defines, I'd do it the way you propose.

Reply:
Right... Let's keep it like this for now.

Reply:
After more thinking I moved all that stuff under one class declaration. So now code looks like this:
```c#
#if ANDROID
        private static MonogamePlatform PlatformGetCurrent()
        {
            return MonogamePlatform.Android;
        }
#endif

#if DESKTOPGL
        private static MonogamePlatform PlatformGetCurrent()
        {
            return MonogamePlatform.DesktopGL;
        }
#endif
```